### PR TITLE
extract translations for level group sublevels

### PR DESCRIPTION
**Pull request by davidsbailey**
Starts https://codedotorg.atlassian.net/browse/PLAT-1751.

This copies the approach in the same file for pulling in bubble choice sublevels. translations for bubble choice sublevels appear to be working end-to-end in CSF 2021 scripts. Here are some debugging notes on that:
* https://levelbuilder-studio.code.org/s/coursea-2021/lessons/13/levels/1/sublevel/1  is a bubble choice sublevel in a translated script
* the translation is located here: https://github.com/code-dot-org/code-dot-org/blob/a09b050d2f890639474c316d10a9ba80a040d29a/dashboard/config/locales/short_instructions.es-MX.json#L3205 
* here you can see it translated in the UI
Screen Shot 2022-07-13 at 10 44 51 AM

so, basically I am naively pattern matching in hopes that this will also get level group sublevels to show up in the short_instructions.*.json files.

## Testing story

**reviewers**: how can I make sure this doesn't break the sync-in?

I plan to verify this by merging the PR, waiting for 1-2 translation syncs, and then verifying that instructions for level group sublevels are starting to show up in places like `dashboard/config/locales/short_instructions.es-MX.json`.  

**reviewers**: are there any other steps you know of that would be necessary to achieve this outcome?

### CODE REVIEW

**Review by smusoke — APPROVED** [review]

**Comment by davidsbailey** [comment]
Thanks for all your help verifying this change with me, Steffan!

**Comment by smusoke** [comment]
This comment is created by running: `bin/i18n/analyze_pr.rb -p 47209 -c`

Top 1/1 files sorted by the number of changes:
- `bin/i18n/sync-in.rb`, 7 changes (7 additions, 0 deletions).

Top 3/3 commits sorted by the number of changes:
- Commit: Merge branch 'staging' of github.com:code-dot-org/code-dot-org into translate-lg-sublevels
  - 11089 changes (6778 additions, 4311 deletions).
  - 300 files changed.
  - File with the most changes: `dashboard/db/schema_cache.yml`, 2343 changes (1222 additions, 1121 deletions).
- Commit: extract translations for level group sublevels
  - 7 changes (7 additions, 0 deletions).
  - 1 files changed.
  - File with the most changes: `bin/i18n/sync-in.rb`, 7 changes (7 additions, 0 deletions).
- Commit: replace sublevels with child_levels
  - 2 changes (1 additions, 1 deletions).
  - 1 files changed.
  - File with the most changes: `bin/i18n/sync-in.rb`, 2 changes (1 additions, 1 deletions).
